README.md

```diff
@@ -211,6 +215,7 @@ validate := validator.New(validator.WithRequiredStructEnabled())
 | uuid_rfc4122 | Universally Unique Identifier UUID RFC4122 |
 | md4 | MD4 hash |
 | md5 | MD5 hash |
+| oci_tag | Open Container Initiative (OCI) Tag |
```

Comment:
Can we maybe leave this and remove all the other formatting changes?

Otherwise, it doesn't feel clear and should probably be cleaned up in a different PR. There are small things like some lost/changed formatting here and there that I would rather not make part of this PR.